src/main.js

```diff
@@ -6,5 +6,17 @@ function setup() {
 }
 
 function draw() {
+  clear();
   game.drawGrid();
+  game.player.draw();
+  game.treasure.draw();
```

Comment:
Since all of these draws are related to the class Game, it would also make sense to create a draw() method inside Game.js that called all of these methods, callind game.draw() in here instead. ✌